dashboards/MySQL_Query_Response_Time.json

```diff
@@ -44,6 +45,7 @@
             "type": "dashboards"
         }
     ],
+    "refresh": "1m",
```

Comment:
@azhebel This was set to false before, do we need to maintain that? @askomorokhov , @rnovikovP ?